### ISSUE TITLE
Package Replication Status Updater uses same session for multiple events

### ISSUE THREAD

**Issue by ghenzler** [Adobe-Consulting-Services/acs-aem-commons#844]
We have configured the Package Replication Status Updater (JcrPackageReplicationStatusEventHandler) which works fine as long as there is not more than one package replicated at a time. We have a special case of mass replication where we use a groovy script to generate packages per content section and replicate them asynchronously (we are not using one package to not let them grow too big). 

Currently this causes many of the following WARNs in the log file:

```
12.11.2016 17:24:22.961 *WARN* [sling-threadpool-e7e460fb-28e9-4f10-9928-15d8de079324-(apache-sling-job-thread-pool)-99-<main queue>(acs-commons/replication/package)] org.apache.jackrabbit.oak.jcr.delegate.SessionDelegate Attempted to perform Setting property [/content/ ... path to page ..... /jcr:content/cq:lastReplicationAction] while thread sling-threadpool-e7e460fb-28e9-4f10-9928-15d8de079324-(apache-sling-job-thread-pool)-104-<main queue>(acs-commons/replication/package) was concurrently writing to this session. Blocked until the other thread finished using this session. Please review your code to avoid concurrent use of a session.
```

The root cause is that the class JcrPackageReplicationStatusEventHandler only uses one admin session for all package replication events it receives:
https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/acs-aem-commons-3.4.0/bundle/src/main/java/com/adobe/acs/commons/replication/status/impl/JcrPackageReplicationStatusEventHandler.java#L212

This should be fixed by creating an admin session for each package replication event.

**Comment by davidjgonzalez** [comment]
@ghenzler thank you! wow... no idea why i ever wrote that :( 

Fixed in #845

**Comment by ghenzler** [comment]
@davidjgonzalez Thanks for the fix :)
